zh/_tldr.md

```diff
@@ -5,39 +5,39 @@
 
 ## 核心原则
 
-* 创建编程规范的目的就是为了保证协作开发的一致性。即使你对本文有不同的意见，也要保证整体开发的一致性。[↩](#section-8)
-* 尽可能让 Sass 代码保持简洁。除非是绝对需要，否则绝没有必要构建复杂的系统。[↩](#section-10)
-* 请记住，有时候保持简洁（KISS，Keep It Simple, Stupid）比减少重复（Don't Repeat Yourself）更重要。[↩](#section-10)
+* 创建编程规范的目的就是为了保证协作开发的一致性。即使你对本文有不同的意见，也要保证整体开发的一致性。[↩](#why-a-styleguide)
```

Comment:
I mean that `section-8` and such should be correct. I’ll double check everything anyway, so feel free to discard your changes on this file. I’ll make it work. ;)